README.md

```diff
@@ -1,3 +1,4 @@
 # algorithms
 Learning how to work through logic problems
 Hello
+World
```

Comment:
For the sake of practice, change this to lowercase `world` instead.

Reply:
Changed `World` to `world`.